pkgs/development/python-modules/dmarc-metrics-exporter/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, poetry-core
+, bite-parser
+, dataclasses-serialization
+, prometheus-client
+, typing-extensions
+, uvicorn
+, xsdata
+, aiohttp
+, pytest-asyncio
+, pytestCheckHook
+, requests
+}:
+
+buildPythonPackage rec {
```

Comment:
Wouldn't `buildPythonApplication` and tracking it outside of pythonPackages be more suitable? It's an application after all.